pkgs/top-level/all-packages.nix

```diff
@@ -22270,6 +22270,8 @@ with pkgs;
 
   lsiutil = callPackage ../os-specific/linux/lsiutil { };
 
+  kaitai_struct = callPackage ../development/compilers/kaitai_struct { };
```

Comment:
A `kaitaistruct` attribute already exists... && same hyphen remark as `pname`

Reply:
This is how other projects named it: https://repology.org/project/kaitai-struct/versions